regxmllib/CMakeLists.txt

```diff
@@ -44,3 +45,17 @@ endforeach( UNIT_TEST_PATH ${UNIT_TESTS} )
 #add_executable(AUIDTest src/test/cpp/com/sandflow/smpte/util/AUIDTest.cpp)
 #target_link_libraries(AUIDTest regxmllibc)
 #add_test(AUIDTest AUIDTest)
+
+file( GLOB_RECURSE HEADER_FILE_LIST ${CMAKE_SOURCE_DIR}/src/main/cpp/com/*.h )
+foreach(HEADER_FILE_PATH ${HEADER_FILE_LIST} )
+    file(RELATIVE_PATH relative_header_path ${CMAKE_SOURCE_DIR}/src/main/cpp/ ${HEADER_FILE_PATH})
+    get_filename_component(relative_path ${relative_header_path} DIRECTORY)
+    install(FILES ${HEADER_FILE_PATH} DESTINATION include/${relative_path})
+endforeach(HEADER_FILE_PATH ${HEADER_FILE_LIST} )
+
+file( GLOB_RECURSE SRC_RESOURCE_LIST ${CMAKE_SOURCE_DIR}/src/main/resources/*.xsd)
+foreach(SRC_RESOURCE_PATH ${SRC_RESOURCE_LIST} )
```

Comment:
reg.xsd is not used by regxmllibc.

Reply:
This installation is just for the user if they want it. We will add it to our code but is not necessary for the source code. We will use the shortened version.

install(DIRECTORY ${CMAKE_SOURCE_DIR}/src/main/resources/ DESTINATION include FILES_MATCHING PATTERN "*.xsd")